packages/block-editor/src/components/inserter/index.js

```diff
@@ -173,6 +173,7 @@ class Inserter extends Component {
 					isAppender={ isAppender }
 					prioritizePatterns={ prioritizePatterns }
 					selectBlockOnInsert={ selectBlockOnInsert }
+					closeQuickInserter={ onClose }
```

Comment:
Suggestion: I think we can use a more general prop name here.

```suggestion
					onClose={ onClose }
```

Reply:
Hi @Mamaduka, thanks for the review. I've implemented the suggested changes in the recent commit.